frontend/src/data/config/MetricConfig.ts

```diff
@@ -252,4 +254,25 @@ export const METRIC_CONFIG: Record<string, VariableConfig[]> = {
       },
     },
   ],
+  health_insurance: [
+    {
+      variableId: "health_coverage",
+      variableDisplayName: "Coverage",
+      metrics: {
+        count: {
+          metricId: "with_health_insurance",
+          fullCardTitleName: "Count of Health Insurance",
+          shortVegaLabel: "Health Insurance",
```

Comment:
maybe "individuals with health insurance" ? this is what will be the label on the disparity bar chart for instance so it will be prefixed with a number i.e. "50 individuals with health insurance"

Reply:
done.